optimum/exporters/openvino/__main__.py

```diff
@@ -232,6 +234,15 @@ def main_export(
         onnx_config_constructor = TasksManager.get_exporter_config_constructor(model=model, exporter="onnx", task=task)
         onnx_config = onnx_config_constructor(model.config)
         models_and_onnx_configs = {"model": (model, onnx_config)}
+        load_in_8bit = model_kwargs.get("load_in_8bit", None)
```

Comment:
what do you think about moving `model_kwargs = model_kwargs or {}` at the beginning of the function to avoid the case where it's `None`